lib/trento/infrastructure/checks/cluster_execution_env.ex

```diff
@@ -9,10 +9,12 @@ defmodule Trento.Infrastructure.Checks.ClusterExecutionEnv do
   require Trento.Enums.Provider, as: Provider
   require Trento.Clusters.Enums.ClusterType, as: ClusterType
   require Trento.Clusters.Enums.FilesystemType, as: FilesystemType
+  require Trento.SapSystems.Enums.EnsaVersion, as: EnsaVersion
```

Comment:
The ensa version we use in the Cluster Execution Env is `Trento.Clusters.Enums.EnsaVersion`, not the SapSystem one